app/assets/javascripts/app/views/contacts_searcher_view.js

```diff
@@ -0,0 +1,122 @@
+app.views.ContactsSearcher = app.views.Base.extend({
```

Comment:
Uhm, the searchbar doesn't search through contacts, it searches through all known (searchable) people. Just `SearchBase` perhaps?